account_invoice_section_sale_order/models/sale_order.py

```diff
@@ -24,7 +24,9 @@ def _create_invoices(self, grouped=False, final=False, date=None):
             ):
                 continue
             sequence = 10
-            move_lines = invoice._get_ordered_invoice_lines()
+            # Because invoices are already created, this would require
+            # an extra read access in order to read order fields.
+            move_lines = invoice.sudo()._get_ordered_invoice_lines()
```

Comment:
can't we avoid calling `sudo` everywhere? It all starts from line 19 where you can call `sudo` at the end of the super call. Plus, can we kindly rename `invoice_ids` to `invoices`? :angel:

Reply:
Updated, can you please approve and merge please ?